Assets/Plugins/Source/Core/EOSManager.cs

```diff
@@ -332,6 +335,11 @@ public void RemoveAuthLogoutListener(IEOSOnAuthLogout authLogout)
                 s_onAuthLogoutCallbacks.Remove(authLogout.OnAuthLogout);
             }
 
+            public void AddApplicationCloseListener(Action listener)
```

Comment:
This should be up on line 322 after the other add functions. And a remove function should be added. Just to maintain consistency with what is already there.

Reply:
I agree.

Reply:
I agree.